packages/core/headings.scss

```diff
@@ -6,13 +6,17 @@
     font-weight: normal;
 }
 
+%bold-font-weight {
+    font-weight: bold;
+}
+
```

Comment:
Jeg ser ikke helt poenget med denne. Er det ikke ryddigere med 
```
$font-weight-bold: bold;

.someClass {
  font-weight: $font-weight-bold;
}
```

Reply:
Enig, var litt monkey see monkey do. 

Hva er fordelen med å ha `font-weight-bold` som alltid vil være bold (basert på navnet) kontra å bare bruke `font-weight: bold` direkte?

Reply:
Det er isåfall at vi tokeniser hva bold er for oss. Så hvis noen vil endre at vi ikke lengre skal si at bold er verdien `bold`, men feks `600`, så er det endring ett sted, ikke 100.

Reply:
Det kan gi mening hvis man sier `$bold-weight: 700`. Det er vekten som er spesifisert i `webfonts`-pakken, og kan være litt forvirrende hvis man ikke er vant til moderne fontvekter. I så fall kan man gjerne gjøre tilsvarende med `$normal-weight: 400`.

Jeg er fortsatt usikker på om det er nødvendig, men skader neppe.